realm/src/main/java/io/realm/RealmQuery.java

```diff
@@ -1131,7 +1120,7 @@ public long count() {
     public RealmResults<E> findAll(String fieldName, boolean sortAscending) {
         TableView tableView = query.findAll();
         TableView.Order order = sortAscending ? TableView.Order.ascending : TableView.Order.descending;
-        Integer columnIndex = columns.get(fieldName);
+        Long columnIndex = columns.get(fieldName);
```

Comment:
Because there is a null check just below it. It is used to indicate not present.